database/helper.js

```diff
@@ -1,12 +1,14 @@
 const { client } = require('./client');
+const { elasticClient } = require('./elasticsearch.js');
+
+
+// INSERTION OPERATIONS
 
 module.exports.addBookingDetail = (obj) => {
   const {
     listingId, searchId, neighbourhood, roomType, nightlyPrices, market, averageRating,
   } = obj;
-  const bookingText = `INSERT INTO listings (listing_id, market, neighbourhood, room_type, review_scores_rating) 
-    VALUES (${listingId}, '${market}', '${neighbourhood}', '${roomType}', ${averageRating}) 
-    ON CONFLICT (listing_id) DO NOTHING`;
+  const bookingText = `INSERT INTO listings (listing_id, market, neighbourhood, room_type, review_scores_rating) VALUES (${listingId}, '${market}', '${neighbourhood}', '${roomType}', ${averageRating}) ON CONFLICT (listing_id) DO NOTHING`;
```

Comment:
Does _listings_ include only booked listings or (eventually) all listings that you consume?

I know for now this is only booked listings, but what about in the future? Or do you think that you will only ever consume information on booked listings?

Reply:
Listings currently only include booked listings. It will likely only consume booked listings in the future since these booked listings come initially from User Behavior first, and then to Inventory. If I want un-booked listings I will have to have a shallow copy of Inventory. Possible, but need quite a bit more time and not likely for MVP.